tavla/app/(admin)/utils/index.ts

```diff
@@ -250,6 +258,23 @@ export function getFormFeedbackForError(
     }
 }
 
+export function fireToastFeedback(
+    addToast: (payload: AddToastPayload | string) => void,
+    result: TFormFeedback | undefined,
+    successMessage: string,
+) {
+    if (result === undefined) {
+        addToast(successMessage)
+    } else if (result.form_type === 'general') {
+        const content =
+            getFormFeedbackForField('general', result)?.feedback || ''
+        addToast({
+            content: content,
+            variant: 'info',
```

Comment:
Bør variant heller kanskje være warning eller negative? typ som denne:
<img width="465" alt="Screenshot 2024-11-25 at 10 53 31" src="https://github.com/user-attachments/assets/d4669ce7-d6e3-4583-b88e-521488bcf53a">
så blir det lettere å se at det er en feil som har skjedd

Reply:
ser ikke ut som at toast-provider støtter rød-farge...... https://design.entur.no/komponenter/feedback/alert#toastprovider

Reply:
Det er også ikke helt meningen å bruke toast for å vise feil i følge designsystemet. Grunner de oppgir: 1) dårlig bruksopplevelse, 2) viktig at man får med seg informasjonen men denne toasten går bort etter x sekunder og vanskelig å se. I utgangspunktet burde vi heller bruke BannerAlertBox eller SmallAletBox (vår FormError-komponent)

Reply:
Med tanke på dette, så er det kanskje unødvendig å skille ut dette til en egen funksjon? Nå vil vi kun fyre av toast om noe faktisk er suksessfullt, også må vi heller vise nærmere handlingen/brukeren (feks ved submittknappen) at noe faktisk ikke funka

Reply:
Der er du god Emilie! FormError overalt isteden da 🆒

Reply:
Å legge til FormError overalt flytter jeg til en separat oppgave. Tenker denne PR-en bør merges inn snart så slipper vi at den ligger så lenge.